Improved the usability for a new user (#334)
Macrometacorp/docs: docs/streams/getting-started-streams.md

## Prerequisites

- Access to a Macrometa account with sufficient permissions to create streams.
- Have your API Key ready. For more information, refer to [Create API Keys] (../account-management/api-keys/create-api-keys)
- Install the appropriate SDK. For more information, refer to [Install SDKs](../sdks/install-sdks.md).

## Get Started with Streams

The basic parts of this quickstart walk you through creating a stream, publishing messages to it, and subscribing to the stream using the [pyC8](https://pyc8.readthedocs.io/en/latest/) and [jsC8](https://www.npmjs.com/package/jsc8) SDKs.

> To get started, create a new javascript (.js) or python (.py) file in your favorite IDE. Then, copy the code block below and paste it into your javascript or python file. With each subsequent step, append the code block to the existing file and run it.

Reviewer: ```suggestion
To get started, create a new JavaScript (.js) or python (.py) file in your favorite IDE. Then, copy the code block below and paste it into your JavaScript or Python file. With each subsequent step, append the code block to the existing file and run it.
```

# Define constants
URL = "gdn.paas.macrometa.io"
GEO_FABRIC = "_system"

> API_KEY = "my API key" # CHANGE THIS TO YOUR API KEY

Reviewer: Please do not use all caps.

## Full Demo File

> It's time to see streams in action! Replace the contents of your .js or .py file from above with the code block below. In your browser, open the GDN console and navigate to the 'streams' tab. Select your recently created stream (c8globals.streamQuickstart) to view the output of the message within the console.

Reviewer: ```suggestion
It's time to see streams in action! 

1. Replace the contents of your .js or .py file from above with the code block below.
2. In your browser, open the GDN console and then click **Streams**.
3. Select your recently created stream (c8globals.streamQuickstart) to view the output of the message within the console.
```

Reviewer: I didn't know you could do this! Neat!

>   - Open two terminal windows and start the program in each window 

Reviewer: Continue the numbered list above.

  - In one terminal, type 'r' to begin listening for messages, while in the other terminal, type 'w' to begin writing messages

>   - Upon each write, you should see the message received in the 2nd terminal window, as well as the message displayed in the GDN console output

Reviewer: ```suggestion
  - Upon each write, you should see the message received in the second terminal window, as well as the message displayed in the GDN console output
```
